### DISCUSSION TITLE
DAC Write equivalent on i2s

### DISCUSSION THREAD

**Discussion by pikkuri14** [pschatzmann/arduino-audio-tools#1811] [Q&A] [answered]
I used built-in esp32 DAC to generate 8bit sample audio, and the audio quite bad. I was planning on using External DAC (maybe PCM5102). I used dac_output_voltage(DAC_CHANNEL_1, 8bitsample) on my program. how can I write that to the external i2s DAC. I dont use i2s ADC input like on the example.

**Comment by pschatzmann** [comment, marked as answer]
Please use the functionality of this library as described in the [Wiki](https://github.com/pschatzmann/arduino-audio-tools/wiki) and the [examples](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-stream).

dac_output_voltage is neither part of my functionality nor should it be used for audio!
